### ISSUE TITLE
@tinymce/tinymce-angular V7.0.0 has abnormal behavior with external scrollbar when pressing Enter.

### ISSUE THREAD

**Issue by Charlie50503** [tinymce/tinymce-angular#362]
## Current Behavior

In the project I'm working on, I'm using the `@tinymce/tinymce-angular` package with version `7.0.0`. I've encountered an issue that is similar to [this GitHub Issue](https://github.com/tinymce/tinymce/issues/8124). When I perform specific actions, the behavior of the editor doesn't match the expected behavior.

## Steps to Reproduce

I've attempted to reproduce this issue, and I've created a simple demo in this [Codesandbox example](https://codesandbox.io/s/tinymce-angular-forked-nwvd7y?file=/src/app/app.component.html) to demonstrate the problem:

1. Enter the page.
2. Scroll the page to the middle.
3. Enter the Tinymce content area and press the "Enter" key repeatedly.
4. When the cursor reaches the bottom of the Tinymce content, press the "Enter" key again and observe if the vertical scroll of the page synchronizes.

You can check out my testing demo through this [example link](https://codesandbox.io/s/tinymce-angular-forked-nwvd7y?file=/src/app/app.component.html).

## Expected Behavior

I expect that the editor's behavior should align with the expected behavior when performing the specific actions, and there should be no abnormal behavior.

## Affected Versions

This issue affects the `@tinymce/tinymce-angular` package with version `7.0.0`. I can't confirm if this issue exists in previous versions of TinyMCE or TinyMCE-Angular because I haven't encountered this problem before, and this is my first time using the package.

## Installed Dependencies

package.json

```json
{
  "name": "test",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "watch": "ng build --watch --configuration development",
    "test": "ng test",
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "^16.0.0",
    "@angular/cdk": "^16.0.3",
    "@angular/common": "^16.0.0",
    "@angular/compiler": "^16.0.0",
    "@angular/core": "^16.0.0",
    "@angular/forms": "^16.0.0",
    "@angular/material": "^16.0.3",
    "@angular/platform-browser": "^16.0.0",
    "@angular/platform-browser-dynamic": "^16.0.0",
    "@angular/router": "^16.0.0",
    "@auth0/angular-jwt": "^5.1.2",
    "@tinymce/tinymce-angular": "^7.0.0",
    "del-cli": "^5.0.0",
    "ng-openapi-gen": "^0.24.1",
    "rxjs": "~7.8.0",
    "tslib": "^2.3.0",
    "zone.js": "~0.13.0"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^16.0.5",
    "@angular/cli": "~16.0.5",
    "@angular/compiler-cli": "^16.0.0",
    "@types/jasmine": "~4.3.0",
    "autoprefixer": "^10.4.14",
    "jasmine-core": "~4.6.0",
    "karma": "~6.4.0",
    "karma-chrome-launcher": "~3.2.0",
    "karma-coverage": "~2.2.0",
    "karma-jasmine": "~5.1.0",
    "karma-jasmine-html-reporter": "~2.0.0",
    "postcss": "^8.4.24",
    "tailwindcss": "^3.3.2",
    "typescript": "~5.0.2"
  }
}

```
## Development Environment

- Operating System: Windows 10
- Browsers:
    - Edge 115
    - Chrome 115

Thank you for your assistance!

**Comment by TinyITAdmin** [comment]
Ref: INT-3218

**Comment by shanmen-tiny** [comment]
Hi @Charlie50503,

Sorry for not getting back to this earlier. I believe this has been resolved in TinyMCE 6.7, see [changelog](https://www.tiny.cloud/docs/tinymce/6/6.7-release-notes/#in-some-circumstances-pressing-the-enter-key-scrolled-the-entire-page). I will close this issue and if the issue persists, feel free to reopen the issue. Thank you.
